src/core/sig_state.ml

```diff
@@ -104,37 +59,34 @@ let add_symbol : sig_state -> sig_symbol -> sig_state * sym =
     | None -> ()
   end;
   let in_scope = StrMap.add x.elt (s, x.pos) ss.in_scope in
-  let pp_hints = remove_pp_hint_eq ss.in_scope x.elt Unqual ss.pp_hints in
-  let pp_hints = SymMap.add s Unqual pp_hints in
+  let pp_hints = SymMap.add s Unqual ss.pp_hints in
```

Comment:
Do we really need this as Not_found => Unqual?

Reply:
Ah indeed, I did not see it. I'll add a note somewhere.

Reply:
Or should we have `notation_of` return a `notation option` with `None` being the current `Unqual`?